### ISSUE TITLE
Make the potential danger of upgrading a channel more prominent in import workflow

### ISSUE THREAD

**Issue by jonboiser** [learningequality/kolibri#5485]
### Observed behavior

When there is a new version of a channel available on studio, there is a UINotification with this copy:

```
'New channel version available. Some of your files may be outdated or deleted.',
```

And an upgrade button. Upgrading a channel can have some unexpected side-effects (as documented in #5268 and #4592)

1. Resources that were once available become unavailable (because the underlying static files might have changed, or the channel itself has changed structurally).
1. Resources orphaned by the upgrade remain on the hard drive, taking up space.
1. Reports and Plan pages Quizzes and Lessons containing resources break.

### Expected behavior

The potential danger of upgrading a Channel is made explicit to users and/or the side-effects mitigated/eliminated.
…

### User-facing consequences

…

### Errors and logs

…

### Steps to reproduce

…

### Context

0.12 develop and all previous versions

**Comment by radinamatic** [comment]
Now that I think of it, I'm really unsure how should I document this...? :confused: 

Is the recommendation _**not**_ to update the channel even if the notification is on? Delete the old version of channel and download again? I don't see users being thrilled about that...

**Comment by jonboiser** [comment]
Just found that deleting a channel does mess up your quiz, leading to bad links on the Coach Pages (reports and plan pages).

(Just realized this isn't the same as an upgrade, but _think_ that upgrading without re-downloading nodes could mess things up in the same way).

**Comment by khangmach** [comment]
Based on the referenced issues, it sounds like we should be fixing/mitigating the following instead of adding new explanations/strings:

>Resources orphaned by the upgrade remain on the hard drive, taking up space.

(upgrading isn't always an additive process and we should be making sure that when upgrading channel versions, resources that aren't existing in the new version of the channel remain deleted off your device). Kind of like github, we should only be showing the latest stuff/diff.

>Reports and Plan pages Quizzes and Lessons containing resources break.

Need to percolate that 404 behavior for all resource breakage in lessons and quizzes

**Comment by jonboiser** [comment]
We should implement these two fixes anyway, but fixing the technical problem of broken coach links doesn't really address the issue that people might want to be informed that upgrading channels can mess things up (and decide _not_ to upgrade, before doing some extra checks).

For example, even if we implemented #5666 Page Not Found pages, for Lessons, etc., it still seems like a problem if a lesson was made, then unexpectedly changed by a channel upgrade, then leading to unexpected problems in the classroom.

**Comment by khangmach** [comment]
Right, we kind of already do that via the string you mention, but could add a bit more detail. 

Possible implementation:
<img width="1024" alt="notif" src="https://user-images.githubusercontent.com/25045166/59872457-a070f400-934e-11e9-9004-d230ee7307b9.png">

**Comment by khangmach** [comment]
I'm actually working on speccing out a banner component at the moment so we can use that to replace the UInotification in this case.

**Comment by radinamatic** [comment]
On the documentation side I'm [adding this warning](https://github.com/learningequality/kolibri-docs/pull/106) (basically, **don't update channels while there are active quizzes and lessons**):

![Selection_129](https://user-images.githubusercontent.com/1457929/60125357-eb926900-978b-11e9-927d-563112fd5f61.png)

Does this seem sufficient for now?

**Comment by khangmach** [comment]
some suggested string change to the first two sentences as they confuse me:

**To prevent loading issues and learners seeing unavailable content, we recommend making any active quizzes and lessons inactive before updating a channel. The location of resources inside of a channel may be changed in an updated version.** It is recommended to schedule and perform...

**Comment by indirectlylit** [comment]
this is being addressed in ongoing 0.13.0 work
